docs/connector/spark/hudi.rst

```diff
@@ -16,21 +16,70 @@
 `Hudi`_
 ========
 
+Apache Hudi (pronounced “hoodie”) is the next generation streaming data lake platform.
+Apache Hudi brings core warehouse and database functionality directly to a data lake.
+
+.. tip::
+   This article assumes that you have mastered the basic knowledge and operation of `Hudi`_.
+   For the knowledge about Hudi not mentioned in this article,
+   you can obtain it from its `Official Documentation`_.
+
+By using Kyuubi, we can run SQL queries towards Hudi which is more convenient, easy to understand,
+and easy to expand than directly using Spark to manipulate Hudi.
+
 Hudi Integration
 ----------------
 
+To enable the integration of kyuubi spark sql engine and Hudi through
+Catalog APIs, you need to:
+
+- Referencing the Hudi :ref:`dependencies`
+- Setting the Spark extension and catalog :ref:`configurations`
+
 .. _dependencies:
 
 Dependencies
 ************
 
+The **classpath** of kyuubi spark sql engine with Hudi supported consists of
+
+1. kyuubi-spark-sql-engine-|release|.jar, the engine jar deployed with Kyuubi distributions
+2. a copy of spark distribution
+3. hudi-spark<spark.version>-bundle_<scala.version>-<hudi.version>.jar, which can be found in the `Maven Central`_
```

Comment:
spark.version and scala.version are usaully ternary, while here might be binary? this may not quite meet the reader's expectation, how about giving an example?